### ISSUE TITLE
New term request - Subcortical maternal complex

### ISSUE THREAD

**Issue by EmilyBowlerBarnett** [geneontology/go-ontology#19581]
Synonyms; SCMC

Definition;
A protein complex comprised of NLRP5, OOEP, TLE6, ZBED3, and KHDC3L, expressed in oocytes and early embryos of mammals, involved in early embryonic cell division processes including the formation of the actin cytoskeleton.

PubMed; 28992324, 18804437

**Comment by pgaudet** [comment]
@bmeldal just pinging you to get your feedback

**Comment by bmeldal** [comment]
How much do we know about Khdc3/KHDC3L? UniProt has a warning that human KHDC3L is not the ortholog to rodent Khdc3 and v.v.. The suppl for 28992324 won't open for me right now to check the interaction data with Khdc3. There is no interaction data for Khdc3 in the main paper. 

28992324 is a bit unclear about whether ZBED3 is a member of the complex of a downstream interactor. I'd be cautious calling it a complex member going by the author's statements.

**Comment by hdrabkin** [comment]
According to MGI, human Q587J8
 (KHDC3L) is the ortholog of mouse Khdc3l.
From our orthology page
http://www.informatics.jax.org/homology.shtml
    MGI uses homology class data from HomoloGene and can support a "many-to-many" relationship between mouse genes and their vertebrate homologs. These data include human, chimpanzee, rhesus macaque, dog, cattle, rat, chicken, western clawed frog, and zebrafish homologs. MGI also uses HGNC/HCOP data for human and mouse orthology.
    Vertebrate Homology Class pages include EntrezGene links, Genetic Location and sequence information. See this complement component Vertebrate Homology Class page for an example.

On what does UniProt make the claim?

**Comment by bmeldal** [comment]
@hdrabkin 

Human KHDC3L https://www.uniprot.org/uniprot/Q587J8:

"Considered by a number of resources to be the ortholog of rodent Khdc3/Filia. However, sequence similarity is low and synteny is not conserved. According to PubMed:17913455, KHDC3L/ECAT1 has been lost in rodents."

Mouse Khdc3 https://www.uniprot.org/uniprot/Q9CWU5:

"Considered by a number of resources to be the ortholog of human KHDC3L/ECAT1. However, sequence similarity is low and synteny is not conserved. According to PubMed:17913455, KHDC3L has been lost in rodents." 

You'll have to ask a UniProt curator about the rational of this comment.

**Comment by hdrabkin** [comment]
I've forwarded this to our Nomen guru for comments.

**Comment by ValWood** [comment]
according to DIOPT
https://www.flyrnai.org/cgi-bin/DRSC_orthologs.pl

human to mouse

high | Yes | Yes | 
Compara, eggNOG, Hieranoid, Homologene, Inparanoid, OrthoFinder, OrthoInspector, orthoMCL, Panther, Phylome, RoundUp, TreeFam

reciprocal best hits.
Supported by *everything*

Low sequence similarity is *never* a reason to discount orthology....many known orthologs are below 20% sequence similarity.
Quite an old paper though....

**Comment by hdrabkin** [comment]
Ongoing investigation here at MGI:
' the MGI orthology Khdc3 mouse to KHDC3L human is based on HomoloGene, which is very outdated -- not by HGNC manual curation itself. I'm going to check with DIOPT since Alliance is showing HGNC orthology of human/mouse and human/rat but I think they are confusing HGNC and HCOP (an issue we dealt with before).
 A few things are in flux:
Alliance orthology data (ReferenceProteome=>Quest for orthologs=>approved algorithms=>DIOPT update is in the works
MGI is transitioning to Alliance orthology data (upcoming software project)

**Comment by ValWood** [comment]
DIOPT is an ortholog aggregator, like HCOP. Both should show the same results really ....
Anyway I would say the orthology is irrefutable based on the agreement between predictors.
Some predictors  often get orthology wrong, but I have never seen ALL predictors get it wrong simultaneously !
Plus it is very very rare for a reciprocal best hit not to be a true ortholog  (I can only think of  a couple of cases, of coiled coil proteins).

**Comment by bmeldal** [comment]
Sounds like the UniProt comment is outdated.

So, someone (not me!) needs to ask UniProt to update the entries.

**Comment by bmeldal** [comment]
Back to the original issue: I think we need more data/publications to know if KHDC3(L) and esp ZBED3 are part of the main complexes.

**Comment by EmilyBowlerBarnett** [comment]
Thanks everyone for your feedback and comments. 

Birgit I agree with the points you've made about the involvement of ZBED3 in this complex, there are no other papers that have data for ZBED3's involvement in the SCMC currently. The wording of the UniProt ZBED3 entries will also need to be changed as a result of this so I'm looking into this already. 
I'm just waiting on feedback from Michele with regards to this and the ortholog caution comment on the KHDC3L and Khdc3 entries. 

I've had a re-read of 28992324 and all of the data is from mouse, so Khdc3 rather than KHDC3L. 
Can I suggest re-wording the description to; 'A protein complex comprised of NLRP5, OOEP, TLE6, and KHDC3, expressed in oocytes and early embryos of mammals, involved in early embryonic cell division processes including the formation of the actin cytoskeleton.'

**Comment by hdrabkin** [comment]
Interestingly, however; HGNC does not have KHDC3, but KHDC3L, and mouse has Khdc3, but not Khdc3l. Our nomen expert is talking with HGNC about possible renames...

**Comment by pgaudet** [comment]
@EmilyBowlerBarnett 

The two papers you cite describe the complex in mouse; is there evidence that it exists in other organisms ? If not the definition should probably be modified to indicate that it is known to exist in mouse (and not in 'mammals'). 

Thanks, Pascale

**Comment by bmeldal** [comment]
Word of caution: If all expts are in mouse but orthologs exist in other rodents or mammals or even further afield the complex may well exist in all of the. Wet lab folk tend to work with what works! And if mouse reagents works... 

We also try to make the definitions function-centric so they don't rely entirely on the component list. It can become a nightmare trying to assess if a complex is appropriate when you find it in a different species compared to the one it was first identified in. In light of no known molecular function I'd focus on the process it is associated with and then say "composed of e.g. xxx in mouse."

When I request UniProt changes I do it through their "contact us" form but I guess you have a different internal process :)

I hope it makes sense.

**Comment by hdrabkin** [comment]
@bmeldal, you're preaching to the choir!
But I think we need something better than 
', involved in early embryonic cell division processes including the formation of the actin cytoskeleton.'

**Comment by bmeldal** [comment]
> @bmeldal, you're preaching to the choir!

LOL, I know, except Emma hasn't yet had the joy of discussing complex definitions ;-)

**Comment by EmilyBowlerBarnett** [comment]
@hdrabkin 
Hi Harold 
Is there any update on the naming/orthology situation with KHDC3 vs KHDC3L?
Thanks

**Comment by hdrabkin** [comment]
@EmilyBowlerBarnett 
I will check with our nomen person.

**Comment by EmilyBowlerBarnett** [comment]
In an effort to resolve the SCMC complex description I’ve recently curated KHDC3/ KHDC3L, and in the process formed the following summary for the GO term. 
The UniProt update to these entries is currently on hold due to the KHDC3/ KHDC3L naming issue.
Revised summary of the SCMC complex;
Protein complex that regulates actin dynamics and plays an essential role for zygotes to progress beyond the first embryonic cell divisions. Required for the formation of F-actin cytoplasmic lattices in oocytes which in turn are responsible for symmetric division of zygotes via the regulation of mitotic spindle formation and positioning.

Localized to the subcortex of oocytes and early embryos, however excluded from cell-cell contact regions after the 2-cell stage.

Comprised of at least NLRP5, OOEP, TLE6, and KHDC3 with evidence of additional SCMC-associated proteins that interact with one or multiple members of the core complex.

Evidence for the complex in humans; 25542835
Evidence for the complex in mice; 28992324, 18804437

**Comment by bmeldal** [comment]
> Comprised of at least NLRP5, OOEP, TLE6, and KHDC3 with evidence of additional SCMC-associated proteins that interact with one or multiple members of the core complex. 

I suggest a slight tweak:

"Comprised IN MOUSE of at least NLRP5, OOEP, TLE6, and KHDC3 (KHDC3L in human)..."

or

"Comprised of at least NLRP5, OOEP, TLE6, and KHDC3/KHDC3L..."

or equivalent amendments if KHDC3 has got orthologs beyond mammals.

**Comment by pgaudet** [comment]
@hdrabkin is this ready to be implemented?

**Comment by EmilyBowlerBarnett** [comment]
I've updated the protein entries for the SCMC components in UniProt to match this, currently retaining the gene name difference for KHDC3/KHDC3L which I can go in and change (or not) once HGNC have decided on the KHDC3/KHDC3L issue.
^^ these updates are not currently live on the website and are waiting to be integrated.

So I'm happy for the summary to go in with Birgit's change of;
'Comprised of at least NLRP5, OOEP, TLE6, and KHDC3/KHDC3L with evidence of additional SCMC-associated proteins that interact with one or multiple members of the core complex.'

**Comment by hdrabkin** [comment]
I'm good with the def.
+id: GO:0106333
+name: subcortical maternal complex
Forgot to add synonym.
Done
